integration/okhttp3/gradle.properties

```diff
@@ -2,14 +2,10 @@ POM_NAME=Glide OkHttp 3.x Integration
 POM_ARTIFACT_ID=okhttp3-integration
 POM_PACKAGING=aar
 
-VERSION_NAME=2.0.0-SNAPSHOT
-VERSION_MAJOR=2
+VERSION_NAME=1.0.0-SNAPSHOT
```

Comment:
Ok, I'll pull version up to :integration like prefix.

To fully understand: Why isn't the glide version used then? If all integrations change together, they might as well change together with Glide version. Would you ever release an integration minor change without a glide minor change?